src/js/utils/events.js

```diff
@@ -250,7 +253,11 @@ export function on(elem, type, fn) {
     return _handleMultipleEvents(on, elem, type, fn);
   }
 
-  const data = DomData.getData(elem);
+  if (!DomData.has(elem)) {
```

Comment:
I see a couple usages of if !has, set, then get. Do you think it's worth a helper?

Reply:
I think that it might now make sense in this case since it is only used in two places.

Reply:
Yeah, you're right. If we end up with this a few more times, then it'll make sense.